publisher_topic: epos_info_topic
service_name: epos_control_service
